apps/web/src/app/api/mcp/documents/route.ts

```
@@ -379,11 +368,27 @@ export async function POST(req: NextRequest) {
+          updates: { content: newContent },
+          updatedFields: ['content'],
+          expectedRevision: typeof page.revision === 'number' ? page.revision : undefined,
+          context: {
+            userId,
+            actorEmail: actorInfo.actorEmail,
+            actorDisplayName: actorInfo.actorDisplayName ?? undefined,
+            metadata: {
+              source: 'mcp',
+              mcpOperation: 'edit-cells',
+              cellsUpdated: cells.length,
+              valuesSet: valueCount,
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)